app/build.gradle

```diff
@@ -64,6 +64,8 @@ dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
     implementation 'junit:junit:4.12'
     implementation 'com.android.support:appcompat-v7:28.0.0'
+    
+
```

Comment:
why this change?

Reply:
It's by mistake. Actually I tried to add a dependency but removed the dependency and the space left by mistake. I'll remove it.

Reply:
Removed in commit 6d3fc439494f7458fa10e77e342790a05c0f8b3e